src/catalogue/items/catalogueItemsLandingPage.component.test.tsx

```diff
@@ -211,57 +178,20 @@ describe('Catalogue Items Landing Page', () => {
     spy.mockRestore();
   });
 
-  it('navigates to catalogue category table view', async () => {
-    createView('/catalogue/item/89');
-    await waitFor(() => {
-      expect(
-        screen.getByRole('link', { name: 'Energy Meters' })
-      ).toBeInTheDocument();
-    });
-
-    const breadcrumb = screen.getByRole('link', {
-      name: 'Energy Meters',
-    });
-
-    await user.click(breadcrumb);
-
-    expect(mockedUseNavigate).toHaveBeenCalledTimes(1);
-    expect(mockedUseNavigate).toHaveBeenCalledWith('/catalogue/5');
-  });
```

Comment:
Was this test moved elsewhere? I think it would have caught the breadcrumbs issue I mentioned. I can't one like below testing the home button either.

Reply:
These these types of test can only be tested in the integration test

Reply:
I will probably forget, might be worth making a note somewhere.

Reply:
This test can be test now with my new implementation